applications/msai.md

```diff
@@ -0,0 +1,290 @@
+|    0a. | License                                | Apache 2.0                                                                                                                                                                                                                                                                                                                               |
+|    0b. | Documentation                          | The extended API will be documented                                                                                                                                                                                                                                                                                                      |
+|    0c. | Testing Guide                          | Each epic describes testing opportunities.  [ 1. Create a health blockchain](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) 2. Create openapi specifcations for reading and writing data to and from the NFT smart contract (Depends on blockchain nft tech used) .                                             |
+|    0d. | Docker                                 | A dockerized setup of the parachain will be provided after several iterations when we have a working secure application against a customised subtrate blockchain with extended off chain worker local storage                                                                                                                            |
+|    0e. | Article                                | Press release will be prepared a few weeks before the official rollout of the network. The link to the article will be sent when ready                                                                                                                                                                                                   | 
+|    1a. | Node Repo                              | Complete the deployment of the basic public chain with example  storage of a native DB format through the off chain worker local storage API.                                                                                                                                                                                            |
+|    2a. | Create New Substrate Pallet            | [ 1. Create demo application](https://trello.com/c/LZ3c4K3T/3-create-basic-parachain-with-useful-api) New parachain with smart contract facility through creation of a dedicated Pallet                                                                                                                                                  |                                                                                                                                                        |
```

Comment:
This sounds like the [contracts pallet](https://paritytech.github.io/substrate/master/pallet_contracts/index.html) to me. Is there any difference to it?

Reply:
Sure it will reuse elements of that but the idea here is to LINK the contract with the off chain storage through a hashing mechanism as descibed both in document and in the references ## References:. I must admit as we are not Substrate experts details may change to satisfy the overall requirement.

Reply:
You can integrate already a hash into any deployed smart contract. Not sure why you need to develop a new contract module for this.

Reply:
@Noc2 Updated application to clarify. You cannot communicate directly from offchain storage with a smart contract that is one chain. This is infrastructure to allow that to happen.

Reply:
Is this point clearer now @takahser I changed the title.
Yes Subtrate and polkadot has the elements. yes anyone wanting todo this can do it but at a significant development cost.
The proposal is based around adding easy to use API so that app developers in the healthcare system CAN use polkadot with much less effort. We ourselves as a startup will be the initial user of any blockchain that integrates this functionality.